src/components/organisms/Menu/index.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import './style.scss';
```

Comment:
Seems wrong order of import here.

Reply:
Should "import './style.scss'" be at the end of the imports? Changed this way.